### PR TITLE
ATL03 Guidelines

### PR DESCRIPTION
**Pull request by smk0033**
Reference ticket: https://github.com/MAAP-Project/maap-documentation/issues/247

Added rendering info, new sections, and adjusted some of the headers and content.

### CODE REVIEW

File: docs/source/science/ATL03/ATL03.ipynb [hunk 1]
```diff
@@ -2,13 +2,57 @@
  "cells": [
```
Comment (jjfrench) [inline docs/source/science/ATL03/ATL03.ipynb:2]: <p>Can you check the concept ID of the collection in which the data is being pulled from? I see that there's two different listings with two different concept IDs in EarthData Search, one from the ?NSIDC server and one on EarthData Cloud</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (smk0033) [reply docs/source/science/ATL03/ATL03.ipynb:2]: Looks like it's using the one from EarthData Cloud

Comment (jjfrench) [reply docs/source/science/ATL03/ATL03.ipynb:2]: If _where_ the data is being pulled from matters, should we change the wording then? Or that portion could be removed?

Comment (smk0033) [reply docs/source/science/ATL03/ATL03.ipynb:2]: I'll probably just go ahead and remove that portion, I don't think it really has to be there

**Review by jjfrench — APPROVED** [review]
Looks good!

**Pull request merged**